src/sentry/tasks/groupowner.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import, print_function
+
+from datetime import timedelta
+
+from django.utils import timezone
+
+from sentry.models.groupowner import GroupOwner, GroupOwnerType
+from sentry.tasks.base import instrumented_task
+from sentry.utils.committers import get_serialized_event_file_committers
+
+
+PREFERRED_GROUP_OWNERS = 3
+PREFERRED_GROUP_OWNER_AGE = timedelta(days=1)
+
+
+@instrumented_task(
+    name="sentry.tasks.process_suspect_commits", default_retry_delay=60 * 5, max_retries=5
+)
+def process_suspect_commits(event, **kwargs):
+    can_process = True
+
+    owners = GroupOwner.objects.filter(
+        group_id=event.group_id, project=event.project, organization=event.project.organization
```

Comment:
The event.project, and event.project.organization relations will emit additional queries here. You might want to use `Project.objects.get_from_cache()` and `project.organization_id` to reduce the number of queries you're doing.

Reply:
Thanks. I didn't know about .get_from_cache()....definitely spent some time looking for that in django's docs 😁